### ISSUE TITLE
Consider using "yaml-ld" for owl_entities

### ISSUE THREAD

**Issue by cmungall** [dosumis/dead_simple_owl_design_patterns#1]
JSON-LD provides a nice way of providing shortforms for IRIs - either OBO-style CURIES or labels. You can add a context at the top of the YAML and this will naturally translate to JSON(LD)

**Comment by dosumis** [comment]
Hmm, I think I may have misunderstood the original ticket (or closed this one by accident).  Seems like we should be doing this though.

**Comment by dosumis** [comment]
Keeping this ticket open as we at least need to provide some guidance for users on how this works in practice.  Should we have some way to refer to the accompanying JSON-LD doc in the patterns?

**Comment by cmungall** [comment]
We seem to have settled on an external prefix file, so no need for embedding context in the file

**Comment by dosumis** [comment]
As we seem to have settled on this for pattern for OBO foundry ontologies

GO: -> http://purl.obolibrary.org/obo/GO_

I feel we should either have guidance on this convention in the OBO extension documentation, or provide a standard prefix file file for OBO on this repo (the pain here being that we'd a mapping for all ontologies in the foundry - so maybe a recipe is better).

**Comment by cmungall** [comment]
I'd favor keeping the spec as simple as possible, and being convention-free. I use explicit prefix maps for all the DPs I directly work on. dosdp-tools has a handy `obo-prefixes` option but this can just be seen as a tool-specific dropin for the prefix map.

**Comment by balhoff** [comment]
I agree with Chris. The `obo-prefixes` option can remain tool-specific. I need to document it better on the dosdp-tools site. If it's seen as promoting laxity I could even remove it, but it's quite handy.
